tools/regression_test.sh

```diff
@@ -367,16 +367,6 @@ function exit_on_error {
   fi
 }
 
-function checkout_rocksdb {
-  echo "Checking out commit $1 ..."
-
-  git fetch --all
-  exit_on_error $?
-
-  git checkout $1
-  exit_on_error $?
-}
-
```

Comment:
I'm assuming this isn't called from anywhere, right? :)

Reply:
Yes, it's dead code.